test/retiarii_test/cgo_mnasnet/base_mnasnet.py

```diff
@@ -40,12 +40,12 @@ def __init__(self, in_ch, out_ch, kernel_size, stride, expansion_factor, skip, b
             # Pointwise
             nn.Conv2d(in_ch, mid_ch, 1, bias=False),
             nn.BatchNorm2d(mid_ch, momentum=bn_momentum),
-            nn.ReLU(inplace=True),
+            nn.ReLU(inplace=False),
```

Comment:
What's the test environment of this test case?

We might be able to add it to our daily / PR test routine.

Reply:
It is used in `test/retiarii_test/cgo_mnasnet/test.py` as an integration test. It requires more than 1 GPUs and takes quite a long time to finish. The current unit tests in `test/ut/retiarii/test_cgo_engine.py` should be enough for test.